packages/TelegramClient-Tests.package/TCTUsersTests.class/instance/mockUserEvent.st

```diff
@@ -0,0 +1,11 @@
+accessing
+mockUserEvent
+
+	^ (TCCEvent newFromTDLibEvent: (Dictionary newFrom: {
+		'@type' -> 'user'.
+		'id' -> self userId.
+		'first_name' -> self userFirstName.
+		'last_name' -> self userLastName.
+		'username' -> self username.
+		'@extra' -> self extra
+	}))
```

Comment:
Could this maybe be migrated to TCTMocks as well? If not, I would at least put it into another category than accessing, given that it does contain some logic

Reply:
moved to TCTMocks